driver-sync/src/test/functional/com/mongodb/client/AbstractChangeStreamsTest.java

```diff
@@ -252,8 +253,8 @@ private JsonPoweredCrudTestHelper createJsonPoweredCrudTestHelper(final MongoCli
 
     @Parameterized.Parameters(name = "{0}: {1}")
     public static Collection<Object[]> data() throws URISyntaxException, IOException {
-        List<Object[]> data = new ArrayList<Object[]>();
-        for (File file : JsonPoweredTestHelper.getTestFiles("/change-streams")) {
+        List<Object[]> data = new ArrayList<>();
+        for (Path file : JsonPoweredTestHelper.getTestFiles("/change-streams", Paths.get("unified"))) {
```

Comment:
I think you can avoid this change if you instead put the unified tests in 

    /unified-test-format/change-streams

instead of

    /change-streams/unified

That's what I plan to do for the new unified crud tests I'm adding in scope of the ticket I'm working on now.

Reply:
I thought about doing this, but `VersionedApiTest` has scenarios in `driver-core/src/test/resources/versioned-api`, not in `driver-core/src/test/resources/unified-test-format/change-streams`. So to be more consistent, I had to use `driver-core/src/test/resources/change-streams/unified`.

Given the fact about `VersionedApiTest`, do you still think I should change the unified `ChangeStreamsTest` runners to take tests from `driver-core/src/test/resources/unified-test-format/change-streams`?

Reply:
Yes, I think we should move all the unified tests under that directory.

Reply:
Great!, this simplifies the changes.

Reply:
Done.